Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM gradle:jdk17 AS builder
+COPY --chown=gradle:gradle . /builder
+WORKDIR /builder
+RUN gradle dependencies
+RUN gradle java-spiffe-helper:assemble -ParchiveClassifier=docker -Pversion=docker
+
+FROM eclipse-temurin:17-jre AS runner
+COPY --from=builder \
+  --chown=nobody:nobody \
+  /builder/java-spiffe-helper/build/libs/java-spiffe-helper-docker-docker.jar /app/java-spiffe-helper.jar
+USER nobody
+ENTRYPOINT ["java", "-jar", "/app/java-spiffe-helper.jar"]
```

Comment:
The `ENTRYPOINT` requires a `--config configfile` parameter. I'm submitting a PR to add a default config file and set a default for this parameter.

Reply:
Yeah, wasn't sure about this one. Effectively the app doesn't start without it, but I didn't want to force users to a location for the file.

Reply:
Probably the way to go, yes! Will add it now so no PR necessary.

Reply:
I believe it would be beneficial to implement a default value for the `--config` parameter. This change would enable the application to run out of the box. I'll be submitting the PR shortly.

Reply:
See #199